hydroflow_datalog_core/src/snapshots/hydroflow_datalog_core__tests__aggregations_and_comments@datalog_program.snap

```diff
@@ -361,34 +361,49 @@ fn main() {
                         >(iter: Iter) -> impl ::std::iter::Iterator<Item = (A, B)> {
                             iter
                         }
+                        #[inline(always)]
+                        /// A: accumulator type
+                        /// T: iterator item type
+                        /// O: output type
+                        fn call_comb_type<A, T, O>(
+                            a: &mut A,
+                            t: T,
+                            f: impl Fn(&mut A, T) -> O,
+                        ) -> O {
+                            f(a, t)
+                        }
```

Comment:
Comment - What does this do? helps type inference?

Reply:
Yeah, just like the non-keyed variants.